storm-client/src/jvm/org/apache/storm/utils/NimbusClient.java

```diff
@@ -96,6 +96,10 @@ public static NimbusClient getConfiguredClientAs(Map<String, Object> conf, Strin
         if (override != null) {
             return new NimbusClient(override);
         }
+        Map<String, Object> fullConf = Utils.readStormConfig();
```

Comment:
I'm not sure this change is related to the origin issue. Could you explain this change?